tap_github/authenticator.py

```diff
@@ -260,22 +260,13 @@ def prepare_tokens(self) -> list[TokenManager]:
         personal_tokens: set[str] = set()
         if "auth_token" in self._config:
             personal_tokens.add(self._config["auth_token"])
-        if "additional_auth_tokens" in self._config:
-            personal_tokens = personal_tokens.union(
-                self._config["additional_auth_tokens"]
-            )
         else:
-            # Accept multiple tokens using environment variables GITHUB_TOKEN*
-            env_tokens = {
-                value
-                for key, value in env_dict.items()
-                if key.startswith("GITHUB_TOKEN")
-            }
-            if len(env_tokens) > 0:
+            # Accept token using environment variable GITHUB_TOKEN
```

Comment:
Could you explain why this change is necessary, seems like the old code supported GITHUB_TOKEN env variable?

Reply:
I took out the `additional_auth_tokens` config which the tap upstream `tap-github` tap supports (for rotating tokens for GitHub.com, to get around API rate limits).

I thought it would be confusing and clash with the GitHub.com + GHE tokens we added support for in `MultiInstanceGitHubAuthenticator`, but happy to add it back in.